app/javascript/article-form/articleForm.jsx

```diff
@@ -139,25 +143,17 @@ export default class ArticleForm extends Component {
     }
   }
 
-  checkContentChanges = previousContent => {
```

Comment:
I can't imagine the case where this is still needed. I believe `isLocalstorageNewer` would handle everything.

Reply:
the only scenario i can see this is collaborative realtime editing but we aren't there yet 🤣